hack/ci/install-cri-o.sh

```diff
@@ -15,10 +15,20 @@
 
 set -euo pipefail
 
-COMMIT_ID=fa262bc87353c45940931402a76f9175a4e01fa6
+COMMIT_ID=a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8
 TAG=v1.24.1
 
-curl "https://raw.githubusercontent.com/cri-o/cri-o/$COMMIT_ID/scripts/get" | bash -s -- -t "$TAG"
+export PATH=$PATH:/usr/local/go/bin
+export GOPATH="$HOME/go"
+export GOBIN="$GOPATH/bin"
+
+BOM_VERSION=v0.3.0
+go install sigs.k8s.io/bom/cmd/bom@$BOM_VERSION
+cp "$GOBIN/bom" /usr/bin
+bom version
+
+# TODO: switch "$COMMIT_ID" back to "$TAG" when CRI-O v1.25.0 is released.
+curl -sSfL --retry 5 --retry-delay 3 "https://raw.githubusercontent.com/cri-o/cri-o/$COMMIT_ID/scripts/get" | bash -s -- -t "$COMMIT_ID"
```

Comment:
I don't quite understand where verification is happening here.

Reply:
It's part of the CRI-O script: https://github.com/cri-o/cri-o/blob/a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8/scripts/get#L138-L174

Reply:
could you add a comment to the code mentioning that both artifact signature and the SBOM are being verified as part of the script? Just so folks know where to look at.

Great stuff!!

Reply:
Yep, added a comment as suggested. :+1: 

It also seems to work now:

#### Ubuntu

```
    default: Welcome to the CRI-O install script!
    default: Using version: a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8
    default: No architecture provided, using: amd64
    default: Checking if all commands are available: curl jq tar
    default: Found cosign, verifying signatures
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.sig
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.cert
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx.sig
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx.cert
    default: Verifying blob cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz
    default: tlog entry verified with uuid: 3916c364efb83458f476875a07f5be43e3e7d37d6ccdedd9e196dab36d18c6f4 index: 2993153
    default: Verified OK
    default: Verifying blob cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx
    default: tlog entry verified with uuid: d30d775155390b54eb744ae7dbd8ddd25e7af9248b1a20612250519e3b63cfa0 index: 2993155
    default: Verified OK
    default: Found bom tool, verifying bill of materials
    default: +-----------------------------------+-------+-----------------------------+----------------+
    default: |             FILENAME              | VALID |           MESSAGE           | INVALID HASHES |
    default: +-----------------------------------+-------+-----------------------------+----------------+
    default: | LICENSE                           | OK    | File validated successfully | -              |
    default: | Makefile                          | OK    | File validated successfully | -              |
    default: | README.md                         | OK    | File validated successfully | -              |
    default: | bin/conmon                        | OK    | File validated successfully | -              |
    default: | bin/crictl                        | OK    | File validated successfully | -              |
    default: | bin/crio                          | OK    | File validated successfully | -              |
    default: | bin/crio-status                   | OK    | File validated successfully | -              |
    default: | bin/crun                          | OK    | File validated successfully | -              |
    default: | bin/pinns                         | OK    | File validated successfully | -              |
    default: | bin/runc                          | OK    | File validated successfully | -              |
    default: | cni-plugins/bandwidth             | OK    | File validated successfully | -              |
    default: | cni-plugins/bridge                | OK    | File validated successfully | -              |
    default: | cni-plugins/dhcp                  | OK    | File validated successfully | -              |
    default: | cni-plugins/firewall              | OK    | File validated successfully | -              |
    default: | cni-plugins/host-device           | OK    | File validated successfully | -              |
    default: | cni-plugins/host-local            | OK    | File validated successfully | -              |
    default: | cni-plugins/ipvlan                | OK    | File validated successfully | -              |
    default: | cni-plugins/loopback              | OK    | File validated successfully | -              |
    default: | cni-plugins/macvlan               | OK    | File validated successfully | -              |
    default: | cni-plugins/portmap               | OK    | File validated successfully | -              |
    default: | cni-plugins/ptp                   | OK    | File validated successfully | -              |
    default: | cni-plugins/sbr                   | OK    | File validated successfully | -              |
    default: | cni-plugins/static                | OK    | File validated successfully | -              |
    default: | cni-plugins/tuning                | OK    | File validated successfully | -              |
    default: | cni-plugins/vlan                  | OK    | File validated successfully | -              |
    default: | cni-plugins/vrf                   | OK    | File validated successfully | -              |
    default: | completions/bash/crio             | OK    | File validated successfully | -              |
    default: | completions/bash/crio-status      | OK    | File validated successfully | -              |
    default: | completions/fish/crio-status.fish | OK    | File validated successfully | -              |
    default: | completions/fish/crio.fish        | OK    | File validated successfully | -              |
    default: | completions/zsh/_crio             | OK    | File validated successfully | -              |
    default: | completions/zsh/_crio-status      | OK    | File validated successfully | -              |
    default: | contrib/10-crio-bridge.conf       | OK    | File validated successfully | -              |
    default: | contrib/crio.service              | OK    | File validated successfully | -              |
    default: | contrib/policy.json               | OK    | File validated successfully | -              |
    default: | etc/10-crun.conf                  | OK    | File validated successfully | -              |
    default: | etc/crictl.yaml                   | OK    | File validated successfully | -              |
    default: | etc/crio-umount.conf              | OK    | File validated successfully | -              |
    default: | etc/crio.conf                     | OK    | File validated successfully | -              |
    default: | install                           | OK    | File validated successfully | -              |
    default: | man/crio-status.8                 | OK    | File validated successfully | -              |
    default: | man/crio.8                        | OK    | File validated successfully | -              |
    default: | man/crio.conf.5                   | OK    | File validated successfully | -              |
    default: | man/crio.conf.d.5                 | OK    | File validated successfully | -              |
    default: +-----------------------------------+-------+-----------------------------+----------------+
    default: /tmp/tmp.6Ifqf6vG7T/cri-o /tmp/tmp.6Ifqf6vG7T /home/vagrant
    default: Installing CRI-O
```
https://github.com/kubernetes-sigs/security-profiles-operator/runs/7447708514?check_suite_focus=true

#### Fedora

```
    default: Welcome to the CRI-O install script!
    default: Using version: a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8
    default: No architecture provided, using: amd64
    default: Checking if all commands are available: curl jq tar
    default: Found cosign, verifying signatures
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.sig
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.cert
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx.sig
    default: Downloading cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx.cert
    default: Verifying blob cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz
    default: tlog entry verified with uuid: 3916c364efb83458f476875a07f5be43e3e7d37d6ccdedd9e196dab36d18c6f4 index: 2993153
    default: Verified OK
    default: Verifying blob cri-o.amd64.a2e57f2ddc89bff6b4f275e6f33b0235c41d64f8.tar.gz.spdx
    default: tlog entry verified with uuid: d30d775155390b54eb744ae7dbd8ddd25e7af9248b1a20612250519e3b63cfa0 index: 2993155
    default: Verified OK
    default: Found bom tool, verifying bill of materials
    default: +-----------------------------------+-------+-----------------------------+----------------+
    default: |             FILENAME              | VALID |           MESSAGE           | INVALID HASHES |
    default: +-----------------------------------+-------+-----------------------------+----------------+
    default: | LICENSE                           | OK    | File validated successfully | -              |
    default: | Makefile                          | OK    | File validated successfully | -              |
    default: | README.md                         | OK    | File validated successfully | -              |
    default: | bin/conmon                        | OK    | File validated successfully | -              |
    default: | bin/crictl                        | OK    | File validated successfully | -              |
    default: | bin/crio                          | OK    | File validated successfully | -              |
    default: | bin/crio-status                   | OK    | File validated successfully | -              |
    default: | bin/crun                          | OK    | File validated successfully | -              |
    default: | bin/pinns                         | OK    | File validated successfully | -              |
    default: | bin/runc                          | OK    | File validated successfully | -              |
    default: | cni-plugins/bandwidth             | OK    | File validated successfully | -              |
    default: | cni-plugins/bridge                | OK    | File validated successfully | -              |
    default: | cni-plugins/dhcp                  | OK    | File validated successfully | -              |
    default: | cni-plugins/firewall              | OK    | File validated successfully | -              |
    default: | cni-plugins/host-device           | OK    | File validated successfully | -              |
    default: | cni-plugins/host-local            | OK    | File validated successfully | -              |
    default: | cni-plugins/ipvlan                | OK    | File validated successfully | -              |
    default: | cni-plugins/loopback              | OK    | File validated successfully | -              |
    default: | cni-plugins/macvlan               | OK    | File validated successfully | -              |
    default: | cni-plugins/portmap               | OK    | File validated successfully | -              |
    default: | cni-plugins/ptp                   | OK    | File validated successfully | -              |
    default: | cni-plugins/sbr                   | OK    | File validated successfully | -              |
    default: | cni-plugins/static                | OK    | File validated successfully | -              |
    default: | cni-plugins/tuning                | OK    | File validated successfully | -              |
    default: | cni-plugins/vlan                  | OK    | File validated successfully | -              |
    default: | cni-plugins/vrf                   | OK    | File validated successfully | -              |
    default: | completions/bash/crio             | OK    | File validated successfully | -              |
    default: | completions/bash/crio-status      | OK    | File validated successfully | -              |
    default: | completions/fish/crio-status.fish | OK    | File validated successfully | -              |
    default: | completions/fish/crio.fish        | OK    | File validated successfully | -              |
    default: | completions/zsh/_crio             | OK    | File validated successfully | -              |
    default: | completions/zsh/_crio-status      | OK    | File validated successfully | -              |
    default: | contrib/10-crio-bridge.conf       | OK    | File validated successfully | -              |
    default: | contrib/crio.service              | OK    | File validated successfully | -              |
    default: | contrib/policy.json               | OK    | File validated successfully | -              |
    default: | etc/10-crun.conf                  | OK    | File validated successfully | -              |
    default: | etc/crictl.yaml                   | OK    | File validated successfully | -              |
    default: | etc/crio-umount.conf              | OK    | File validated successfully | -              |
    default: | etc/crio.conf                     | OK    | File validated successfully | -              |
    default: | install                           | OK    | File validated successfully | -              |
    default: | man/crio-status.8                 | OK    | File validated successfully | -              |
    default: | man/crio.8                        | OK    | File validated successfully | -              |
    default: | man/crio.conf.5                   | OK    | File validated successfully | -              |
    default: | man/crio.conf.d.5                 | OK    | File validated successfully | -              |
    default: +-----------------------------------+-------+-----------------------------+----------------+
    default: /tmp/tmp.fa8xxGUCeQ/cri-o /tmp/tmp.fa8xxGUCeQ /home/vagrant
    default: Installing CRI-O
```
https://github.com/kubernetes-sigs/security-profiles-operator/runs/7447708254?check_suite_focus=true